internal/pkg/cli/flag.go

```diff
@@ -137,7 +137,7 @@ const (
 	envFlagShort      = "e"
 	typeFlagShort     = "t"
 	workloadFlagShort = "w"
-	previousFlagShort = "p"
+	// previousFlagShort = "p"
```

Comment:
We can comment comment [previous flag](https://github.com/aws/copilot-cli/blob/d4c353e0ddbcd0e1c644780391b01b6024d1d5ac/internal/pkg/cli/flag.go#L41) too.

Reply:
it's used in https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/cli/svc_logs.go#L282 too, so we don't need to comment it out

Reply:
Yeah that's just a validation but yeah we can keep it anyway.